### ISSUE TITLE
How to customise ASP.NET Core Grpc.Core.RpcException for correct Sentry issue tracking

### ISSUE THREAD

**Issue by MaxVynohradov** [getsentry/sentry-dotnet#2896]
Please mark the type framework used:

- [X ] ASP.NET Core

Please mark the type of the runtime used:

- [X] .NET Core

* Version:

Please mark the NuGet packages used:

- [ X] Sentry
- [ X] Sentry.AspNetCore

* Version:
        <PackageReference Include="Sentry" Version="3.41.0" />
        <PackageReference Include="Sentry.AspNetCore.Grpc" Version="3.41.0" />

We are using Sentry to monitor errors/issues on the ASP.NET Core gRPC service.

The issue is that the server returns Grpc.Core.RpcException for ALL exceptions, just with different messages and traces. As a result in Sentry, we have just one issue with hundreds of different error.

In other words - we want to split this somehow.

**Comment by bitsandfoxes** [comment]
This sounds like something we should improve on the SDK side.
But to unblock yourself you have a couple of options:
- It looks like the default fingerprint grouping does not work well in your case. We have some [docs](https://docs.sentry.io/product/data-management-settings/event-grouping/) here. You can modify these for your project or use [custom fingerprinting](https://docs.sentry.io/platforms/dotnet/usage/sdk-fingerprinting/) that can be implemented in the [BeforeSend](https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/configuration/filtering/)
